.github/workflows/catalog-updates-check.yaml

```diff
@@ -0,0 +1,60 @@
+name: Check for unsafe catalog updates in latest-dev
+"on":
+  pull_request:
+    types: [opened, synchronize, reopened, edited]
+jobs:
+  # Check if the PR has updated latest-dev. If a catalog update is required, but latest-dev
+  # is unchanged, this will be caught in the update tests so we don't need to double check here.
+  # Otherwise, read latest-dev contents to determine if there are column added or dropped from
+  # catalog tables. A "safe" modification should rebuild the table to ensure consistent attribute
+  # numbers across versions, instead of simply doing ALTER TABLE ... ADD/DROP COLUMN
+  check_latest_dev_correctly_updated:
+    name: Check that latest-dev.sql was properly updated by PR
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+          fetch-depth: 0
+      - name: Check if latest-dev was modified
+        shell: bash --norc --noprofile {0}
+        id: check_updated
+        run: |
+          file="sql/updates/latest-dev.sql"
+          updated=0
+          # diff=$(git diff --name-status origin/main | grep "${file}")
```

Comment:
Nit: Using `gh` (which is available in [all GitHub runners](https://docs.github.com/en/actions/using-workflows/using-github-cli-in-workflows)) might be a bit easier:

```
gh pr view $PR_NUMBER --json files --jq '.files[] | select(.path == "sql/updates/latest-dev.sql") | .path'

...
## And add this to the step
env:
  PR_NUMBER: ${{ github.event.number }}
```

Reply:
updated